antarest/study/business/st_storage_manager.py

```diff
@@ -445,29 +445,30 @@ def update_storage(
         values = new_config.dict(by_alias=False)
         return StorageOutput(**values)
 
-    def delete_storage(
+    def delete_storages(
         self,
         study: Study,
         area_id: str,
-        storage_id: str,
+        form: List[str],
```

Comment:
You can also use `Sequence` instead of `List`. Using `Sequence[str]` as a type for a parameter indicates that the collection will not be modified, as `Sequence` represents an immutable sequence.